python/test/t_PostAnalyticalControlledImportanceSampling_std.py

```
@@ -52,3 +52,36 @@
+        result_form = algo.getResult()
+
+        algo = ot.PostAnalyticalControlledImportanceSampling(result_form)
+        algo.setMaximumOuterSampling(1000)
+        algo.setBlockSize(10)
+        algo.setMaximumCoefficientOfVariation(0.001)
+        algo.run()
+        if i == 0:
+            ref = threshold
+        else:
+            ref = 1.0 - threshold
+        assert_almost_equal(algo.getResult().getProbabilityEstimate(), ref, 1e-2, 1e-2)
```

Ruff (0.15.17)

[error] Undefined name `assert_almost_equal`

(F821)